Common/NearestNeighbour/src/main/java/org/tribuo/common/nearest/KNNTrainer.java

```diff
@@ -86,7 +89,7 @@ public DistanceType getDistanceType() {
     private Distance distance;
 
     @Config(description = "The distance function used to measure nearest neighbours.")
-    private DistanceType distType;
+    private DistanceType distType = DistanceType.L2;
```

Comment:
Same as HDBSCAN, this means old config files which set `distance` won't work because both fields will be set.